test/unit/plugin/v2/api_cli_test.rb

```diff
@@ -0,0 +1,4 @@
+# you can call Inspec.plugin(2, :cli) and get the CLI plugin base class
```

Comment:
Do we need this file?

Reply:
It was just my way of keep notes prior to a branching strategy bing determined. I'll remove them, as these (implemented) tests will come in on another branch.